openc3/spec/packets/parsers/xtce_parser_spec.rb

```diff
@@ -294,6 +294,190 @@ def add_cmd_packet_item(tf, packet, item)
+          file.puts "            <xtce:Term coefficient=\"1.0\" exponent=\"0\"/>"
+          file.puts "          </xtce:Polynomial>"
+          file.puts "        </xtce:CRC>"
+          file.puts "      </xtce:ErrorDetectCorrect>"
+          file.puts "      <xtce:SizeInBits>"
+          file.puts "          <xtce:FixedValue>16</xtce:FixedValue>"
+          file.puts "      </xtce:SizeInBits>"
+          file.puts "    </xtce:BinaryDataEncoding>"
+          file.puts "  </xtce:BinaryArgumentType>"
+          file.puts "  <xtce:BinaryArgumentType name=\"B_Type\" shortDescription=\"Packet CRC\">"
+          file.puts "    <xtce:UnitSet/>"
+          file.puts "    <xtce:BinaryDataEncoding>"
+          file.puts "      <xtce:ErrorDetectCorrect>"
+          file.puts "        <xtce:Parity>"
+          # TODO: Not sure what this looks like
+          file.puts "        </xtce:Parity>"
+          file.puts "      </xtce:ErrorDetectCorrect>"
+          file.puts "      <xtce:SizeInBits>"
+          file.puts "          <xtce:FixedValue>16</xtce:FixedValue>"
+          file.puts "      </xtce:SizeInBits>"
+          file.puts "    </xtce:BinaryDataEncoding>"
+          file.puts "  </xtce:BinaryArgumentType>"
+          file.puts "</xtce:ArgumentTypeSet>"
+          file.puts "<xtce:MetaCommandSet>"
+          file.puts "  <xtce:MetaCommand name=\"CMD_Base\" abstract=\"true\">"
+          file.puts "    <xtce:ArgumentList>"
+          file.puts "      <xtce:Argument name=\"A\" argumentTypeRef=\"A_Type\"/>"
+          file.puts "      <xtce:Argument name=\"B\" argumentTypeRef=\"B_Type\"/>"
+          file.puts "    </xtce:ArgumentList>"
+          file.puts "    <xtce:CommandContainer name=\"CMD_CommandContainer\">"
+          file.puts "      <xtce:EntryList>"
+          file.puts "        <xtce:ArgumentRefEntry argumentRef=\"A\"/>"
+          file.puts "        <xtce:ArgumentRefEntry argumentRef=\"B\"/>"
+          file.puts "      </xtce:EntryList>"
+          file.puts "    </xtce:CommandContainer>"
+          file.puts "  </xtce:MetaCommand>"
+          file.puts "  <xtce:MetaCommand name=\"CMD\" shortDescription=\"CMD description\">"
+          file.puts "    <xtce:BaseMetaCommand metaCommandRef=\"CMD_Base\"/>"
+          file.puts "  </xtce:MetaCommand>"
+          file.puts "</xtce:MetaCommandSet>"
+        end
+
+        @pc.process_file(tf.path, 'TGT')
+        packet = @pc.commands['TGT']['CMD']
+        expect(packet).to_not be_nil
+        expect(packet.get_item('A').bit_offset).to eql 0
+        expect(packet.get_item('A').bit_size).to eql 16
+        expect(packet.get_item('A').data_type).to eql :BLOCK
+        # TODO: Size is 16 bits but we're doing a polynomial write which is a 64 bit float
+        # So not sure what's supposed to happen to convert a 64bit float to a 16 bit value
+        expect(packet.get_item('A').write_conversion.class).to eql PolynomialConversion
```

Comment:
CRC polynomials are not polynomial conversions.  They define one of the parameters necessary for the CRC algorithm.